React/Views/RefreshControl/RCTRefreshControl.m

```diff
@@ -49,6 +49,16 @@ - (void)layoutSubviews
   _isInitialRender = false;
 }
 
+- (void)didMoveToWindow
+{
```

Comment:
I think we should call `[super didMoveToWindow];` here as well?

Reply:
Good catch, added it.